index.html

```diff
@@ -252,6 +379,7 @@ <h3>Profesional Art Printing Data</h3>
       <div class="about_container">
         <h3 class="about_me_title">About me</h3>
         <div class="about_info">
+          <h3 class="about_me_title-desktop">About me</h3>
```

Comment:
- [ ] You project looks great, exact copy of Figma, it is impressive! I only noticed that languages list in `About me` section are not aligned to the left on desktop screens: 

Your design  | Figma design
------------- | -------------
<img width="819" alt="Screenshot 2022-03-24 at 16 29 58" src="https://user-images.githubusercontent.com/77038610/159916477-a3964d7b-5c30-46cf-9681-12e721b7e884.png"> | <img width="699" alt="Screenshot 2022-03-24 at 16 29 34" src="https://user-images.githubusercontent.com/77038610/159916408-d1fda71d-caf9-4ebf-9045-5246a14e9922.png">

- [ ] Please also make sure to give more space on the left and right sides.